WebRoot/css/zm.css

```diff
@@ -1493,19 +1542,26 @@ INPUT[type="text"].search_input-expanded:focus {
 
 /* Styles to show addresses in bubbles in input fields */
 
-.addrBubbleContainer,
+.addrBubbleContainer {
+	@BottomSeparator@
+	@TextCursor@
+	position:static;
+	overflow:visible;
+	background-color:white;
+	min-height:3rem;
+}
+
 .attBubbleContainer {
-	@InputBorder@
 	@TextCursor@
 	position:static;
 	overflow:visible;
 	background-color:white;
-	min-height:2rem;
+	min-height:3rem;
 }
 
 .attBubbleContainer.ZFocused,
 .addrBubbleContainer.ZFocused {
-    @InputField-focused@
+    @AppViewInputField-focused@
```

Comment:
specifically added property of input field in ZmComposeView . Ux asked not to have a yellow or any bg color on hover for to/cc/bcc fields which is default for the inputs in the app.